lesson_10/homework_10.py

```diff
@@ -0,0 +1,49 @@
+"""Homework 11.
+
+Ваша команда та ви розробляєте систему входу для веб-додатка,
+і вам потрібно реалізувати тести на функцію для логування
+подій в системі входу.
+Дано функцію, напишіть набір тестів для неї.
+"""
+
+import logging
+from logging.handlers import RotatingFileHandler
+
+# Logger settings
+logger = logging.getLogger('log_event')
+logger.setLevel(logging.INFO)
+
+# Rotating handler
+rotating_handler = RotatingFileHandler(
+    filename='login_system.log',
+    mode='a',
+    maxBytes=10000,
+    backupCount=5,
+)
+log_formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
+rotating_handler.setFormatter(log_formatter)
+logger.addHandler(rotating_handler)
```

Comment:
It's not good practice to modify code from developers))
This time OK let's close eyes - but for next time just accept input code as is